tests/php/ExternalLinks/ExternalLinksTest.php

```diff
@@ -125,6 +127,32 @@ public function testArchivedPagesAreHiddenFromReport()
         $this->assertEquals(3, BrokenExternalLinksReport::create()->sourceRecords()->count());
     }
 
+    public function testMissingPageGracefullyHandled(): void
+    {
+        // Run link checker to generate BrokenExternalPageTrackStatus
+        $task = CheckExternalLinksTask::create();
+        $task->runLinksCheck(PolyOutput::create(PolyOutput::FORMAT_ANSI, PolyOutput::VERBOSITY_QUIET));
+        // Simulate an old status that didn't complete, though was later reused
+        // also simluate a page it was tracking was deleted
+        $status = BrokenExternalPageTrackStatus::get_latest();
+        $status->Status = 'Running';
+        $status->write();
+        $pageID = BrokenExternalPageTrack::get()->max('ID') + 1;
+        $pageTrack = BrokenExternalPageTrack::create();
```

Comment:
Use `new` instead of `::create()` as per https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#object-instantiation

> For unit tests, always use the `new` keyword when instantiating objects unless you are explicitly testing the injector logic.

Reply:
Updated